content/applications/productivity/discuss/overview/get_started.rst

```diff
@@ -1,71 +1,78 @@
 
-Choose your notifications preference
-====================================
+Choose notifications preference
+===============================
 
-Access your *Preferences* and choose how you would like your notifications to be handled.
+Access *preferences* by navigating to :menuselection:`Settings app --> Users --> User -->
+Preferences tab`. Then choose either :guilabel:`Handle by Emails` or :guilabel:`Handle in Odoo` for
+how the user's notifications should be handled.
 
-.. image:: get_started/change_preferences.png
+.. image:: get_started/preferences-user.png
    :align: center
    :alt: View of the preferences page for Odoo Discuss
 
-| By default, the field is set as *Handle by Emails* making messages, notes, and notifications where
-  you were mentioned or that you follow, to be sent through email. By choosing *Handle in Odoo*,
-  they are shown in the *Inbox*.
-| Messages can then be *Marked as Todo*, *Replied*, or *Marked as Read*.
+By default, the field is set as :guilabel:`Handle by Emails`. With this setting enabled, a
+notification email will be sent by Odoo every time a message is sent, a note is sent with an '@'
+mention, or a notification is sent for a record that the user follows.
 
-.. image:: get_started/inbox_actions.png
+By choosing :guilabel:`Handle in Odoo`, the above notifications are shown in the *Dicuss* app's
+:guilabel:`Inbox`. Messages can have the following actions taken on them: respond with an emoji by
+clicking :guilabel:`Add a Reaction`, or reply to the message by clicking on :guilabel:`Reply`.
+Additional actions may include starring the message by clicking :guilabel:`Marked as Todo`, or
+pinning the message by selecting :guilabel:`Pin` or even mark the message as unread by selecting
+:guilabel:`Marked as unread`.
+
+.. image:: get_started/reactions-discuss.png
    :align: center
    :alt: View of an inbox message and its action options in Odoo Discuss
 
-The messages tagged as *Mark as Todo* are also shown in *Starred*, while the ones *Marked as Read*
-are moved to *History*.
+Clicking :guilabel:`Mark as Todo` on a message causes it to appear on the :guilabel:`Starred` page,
+while clicking :guilabel:`Mark as Read` moves the message to :guilabel:`History`.
 
 .. image:: get_started/starred_messages.png
    :align: center
    :alt: View of messages marked as todo in Odoo Discuss
 
-Start Chatting
+Start chatting
 ==============
 
-The first time you log in to your account, OdooBot sends you a message asking for permission to
-receive desktop notifications to chats. If accepted, you receive push notifications to the messages
-sent to you despite of where you are in Odoo.
+The first time a user logs in to their account, OdooBot sends a message asking for permission to
+send desktop notifications for chats. If accepted, the user will receive push notifications on their
+desktop for the messages they receive, regardless of where the user is in Odoo.
 
 .. image:: get_started/odoobot_push.png
```

Comment:
This image is too wide at 944px, can you please resize it to be 768px wide (933px max), also the image filename contains an underscore, please change it to hyphens